src/face_op.cpp

```diff
@@ -130,7 +130,7 @@ void Manifold::Impl::Face2Tri(const Vec<int>& faceEdge,
     const PolygonsIdx polys =
         Face2Polygons(halfedge_.cbegin() + faceEdge[face],
                       halfedge_.cbegin() + faceEdge[face + 1], projection);
-    return TriangulateIdx(polys, precision_);
+    return TriangulateIdx(polys, tolerance_);
```

Comment:
I think `Face2Tri` should be using uncertainty instead of tolerance. Otherwise it will generate negative area triangles with a thickness of tolerance, which I don't think we want.

Reply:
Yeah, thinking about it, tolerance should only be used when we are doing simplification.

Reply:
Also, maybe rename `uncertainty` to `epsilon`? It would fit with our algorithm docs better.